hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/SetNodeOperationalStateCommandHandler.java

```diff
@@ -102,6 +102,26 @@ public void handle(SCMCommand<?> command, OzoneContainer container,
       // TODO - this should probably be raised, but it will break the command
       //      handler interface.
     }
+
+    // Stop DiskBalancerService if the state is DECOMMISSIONING or ENTERING_MAINTENANCE
```

Comment:
Please follow the same way as "replicationSupervisor.accept(state)" to handle the state change for disk balancer.